src/libraries/System.Text.Json/tests/Serialization/SerializationWrapper.cs

```diff
@@ -62,7 +62,7 @@ protected internal override string Serialize(object value, Type inputType, JsonS
                     using var stream = new MemoryStream();
                     await JsonSerializer.SerializeAsync(stream, value, inputType, options);
                     return Encoding.UTF8.GetString(stream.ToArray());
-                }).Result;
+                }).GetAwaiter().GetResult();
```

Comment:
This change so we throw an actual exception, correct? (not AggregateException).

Reply:
Yes.